src/common/Screen.js

```diff
@@ -85,6 +87,7 @@ class Screen extends PureComponent<Props> {
     search: false,
     autoFocus: false,
     searchBarOnChange: (text: string) => {},
+    shouldShowLoadingBanner: true,
```

Comment:
Let's squash this commit with the one after it, which passes false for this in several places. That way we don't have an intermediate state where the banner appears in some places where it doesn't make sense.

Both changes are small, so that doesn't make it a lot to read. If this were a more complicated transition, there are other ways it could be split up -- e.g., add the option default-off, then add explicit opt-outs one by one or in groups of related callsites, then flip the default. But happily none of that is needed here.